data-raw/qc-markdowns/seine-snorkel-data/feather-river/combined_feather_snorkel_qc.Rmd

```diff
@@ -0,0 +1,143 @@
+---
+title: "Combine Feather River Snorkel Data"
+author: "Liz Stebbins"
+date: "5/2/2024"
+output: rmarkdown::github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE, fig.width=15, fig.height=10)
+library(tidyverse)
+library(googleCloudStorageR)
+library(knitr)
+color_pal <- c("#9A8822",  "#F8AFA8", "#FDDDA0", "#74A089", "#899DA4", "#446455", "#DC863B", "#C93312")
+
+gcs_auth(json_file = Sys.getenv("GCS_AUTH_FILE"))
+gcs_global_bucket(bucket = Sys.getenv("GCS_DEFAULT_BUCKET"))
+```
+
+## Adult Redd Data Standardization
+
+FlowWest received snorkel data for Feather River in two separate Access databases: one for pre-2004 data and one for 2004-2020 data. These are processed in separate markdowns.
+  
+```{r, message = FALSE, warning = FALSE}
+gcs_get_object(object_name = 
+                 "juvenile-rearing-monitoring/seine-and-snorkel-data/feather-river/data-raw/pre_2004_snorkel.csv",
+               bucket = gcs_get_global_bucket(),
+               saveToDisk = here::here("data-raw", "qc-markdowns", "seine-snorkel-data", "feather-river", "snorkel_pre_2004.csv"),
+               overwrite = TRUE)
+gcs_get_object(object_name = 
+                 "juvenile-rearing-monitoring/seine-and-snorkel-data/feather-river/data-raw/post_2004_snorkel.csv",
+               bucket = gcs_get_global_bucket(),
+               saveToDisk = here::here("data-raw", "qc-markdowns", "seine-snorkel-data", "feather-river", "snorkel_post_2004.csv"),
+               overwrite = TRUE)
+
+snorkel_pre_2004 <- read_csv(here::here("data-raw", "qc-markdowns", "seine-snorkel-data", "feather-river", "snorkel_pre_2004.csv"))
+snorkel_post_2004 <- read_csv(here::here("data-raw", "qc-markdowns", "seine-snorkel-data", "feather-river", "snorkel_post_2004.csv"))
+
+```
+
+## Standard format for Adult All Redd Data
+
+
+  
+## Read in data {.tabset}
+  
+Below we read in the snorkel data from the different databases and rename or select columns so that we can join all the monitoring datasets together in the section below.
+
+### Pre-2004
+
+#### Columns Removed
+
+- `flow` and `adj_flow` - sparsely recorded and values not on the same scale as `river_flow`
+- `fish_depth` seemed redundant with `river_depth`
+- `max_fork_length` seemed redundant with `fork_length`
+
+columns created/renamed:
+- `river_flow` renamed to `velocity`
```

Comment:
These values look pretty high (go up to 10000) so they seem like flows and not velocity measures to me. Please confirm and rename.